cmd/swarm/config_test.go

```diff
@@ -250,8 +238,10 @@ func TestConfigCmdLineOverrides(t *testing.T) {
 		fmt.Sprintf("--%s", EnsAPIFlag.Name), "",
 		fmt.Sprintf("--%s", utils.DataDirFlag.Name), dir,
 		fmt.Sprintf("--%s", utils.IPCPathFlag.Name), conf.IPCPath,
-		"--verbosity", fmt.Sprintf("%d", *testutil.Loglevel),
+		fmt.Sprintf("--%s", SwarmSwapPaymentThresholdFlag.Name), strconv.Itoa(swap.DefaultPaymentThreshold),
```

Comment:
Why are you removing the verbosity flag? I don't think that is correct.

Reply:
Indeed not correct. Thanks for spotting it!